controllers/api.controller.js

```diff
@@ -4,7 +4,7 @@ exports.getApi = (req, res, next) => {
   return fs
     .readFile(`${__dirname}/../endpoints.json`, { encoding: "utf8" })
     .then((data) => {
-      res.status(200).send(JSON.parse(data));
+      res.status(200).send({ data: JSON.parse(data) });
```

Comment:
The key of data doesnt quite make sense/is a little vague for the user with what they're looking at - 
this is a json list of your api routes.
Routes, or paths make more sense, for pathInfo, or apiInfo
Technically it would be `information`, as its been processed, not `data`

Reply:
Alright, I change it into `pathInfo`